tests/foreman/cli/test_repository.py

```diff
@@ -246,31 +246,20 @@ def test_positive_create_6(self, name):
         @BZ: 1103944
 
         """
-
-        # Make a new gpg key
         new_gpg_key = make_gpg_key({'organization-id': self.org['id']})
-
         new_repo = self._make_repository({
             u'name': name,
             u'gpg-key': new_gpg_key['name'],
         })
 
-        # Fetch it again
-        result = Repository.info({'id': new_repo['id']})
```

Comment:
`self._make_repository` uses a CLI factory to create the repository and as factories already grabs the information to return I'm dropping this info to avoid running info two times.